sonaranalyzer-dotnet/its/regression-test.ps1

```diff
@@ -30,6 +30,12 @@ function Test-SonarAnalyzerDll {
     }
 }
 
+function Test-SonarAnalyzerCFGDll {
```

Comment:
Please update this to `Test-FileExists([string]$path)` to unify Test-SonarAnalyzerDll and Test-SonarAnalyzerCFGDll and decouple the fixed paths for test and error message.
I suggest to move it to `\scripts\utils.ps1` that is already included. It could be used on other places.